persist-admin-notices-dismissal.php

```diff
@@ -100,9 +100,13 @@ public static function dismiss_admin_notice() {
 		 * @return bool
 		 */
 		public static function is_admin_notice_active( $arg ) {
-			$array       = explode( '-', $arg );
-			$length      = array_pop( $array );
-			$option_name = implode( '-', $array );
+			if (strpos($arg,'-')) {
```

Comment:
@afragen it seems the `data-` requirement isn't needed. Could you update the readme to reflect this. This line to be specific.

```
Note: the data-dismissible attribute must **have a unique hyphen separated text prefixed by data-** which will serve as the key or option name used by the Options API to persist the state to the database. Don't understand, see the following examples.
```

Might want to test if it works before effecting the above change 😄